.github/workflows/test.yml

```
@@ -0,0 +1,86 @@
+name: Test Workflow
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, edited, labeled, workflow_dispatch]
+  workflow_dispatch:
+      inputs:
+        dry-run:
+          description: 'Run tests but do not create issues {true,false}'
+          required: true
+          default: 'true'
+        notify-id:
+          description: 'Issue notification {github id}'
+          required: false
+          default:
+        vendor-type:
+          description: 'Vendor type {all,partner,redhat,community}'
+          required: true
+          default: 'all'
+        software-name:
+          description: 'Software Name'
+          required: true
+        software-version:
+          description: 'Software Version'
+          required: true
+
+
+jobs:
+  workflow-test:
+    name: Workflow Test
+    runs-on: ubuntu-20.04
+    if: github.event.pull_request.draft == false
+
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Python 3.x Part 1
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+
+      - name: Set up Python 3.x Part 2
+        run: |
+          # set up python requirements and scripts
+          python3 -m venv ve1
+          cd scripts && ../ve1/bin/pip3 install -r requirements.txt && cd ..
+          cd scripts && ../ve1/bin/python3 setup.py install && cd ..
+
+      - name: Check Request
+        id: check_request
+        run: |
+          # check if workflow testing should run.
+          echo "Request type: $GITHUB_EVENT_NAME"
+          if [ "$GITHUB_EVENT_NAME" == "pull_request" ]; then
+              echo "[INFO] check if PR contains only workflow changes and user is authorized"
+              ve1/bin/check-pr-for-ci --verify-user=${{ github.event.pull_request.user.login }} --api-url=${{ github.event.pull_request._links.self.href }}
+          else
+              echo "[INFO] manual invocation - check if user is authorized"
+              ve1/bin/check-pr-for-ci --verify-user=${{ github.actor }}
+          fi
+
+      - name: Test CI Workflow
+        id: test_ci_workflow
+        if: ${{ steps.check_request.outputs.run-tests == 'true'}}
+        env:
+          CLUSTER_TOKEN: ${{ secrets.CLUSTER_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          DRY_RUN: = ${{ github.event.inputs.dry-run }}
+          VENDOR_TYPE: = ${{ github.event.inputs.vendor-type }}
+          NOTIFY_ID: = ${{ github.event.inputs.notify-id }}
+          SOFTWARE_NAME: = ${{ github.event.inputs.software-name }}
+          SOFTWARE_VERSION: = ${{ github.event.inputs.software-version }}
```

```suggestion
          SOFTWARE_VERSION: ${{ github.event.inputs.software-version }}
```

Same for above variables..